src/web/command_control/client/components/TaskSettingsPanel.tsx

```diff
@@ -272,7 +272,7 @@ export function TaskSettingsPanel(props: Props) {
 
         // Change to NONE
         if (newTaskType == TaskType.NONE) {
-            props.onChange(null)
+            props.onChange(undefined)
```

Comment:
The taskChangeTester() function in CommandControl gets pretty messy when testing for both a TaskType of 'Undefined' (the value before the task is set) and 'Null' (the value if the task is changed back to 'None'). Changing this line from 'null' to 'undefined' makes it so that when the user changes the task to 'None', the value is represented as 'Undefined' again. 
I can change it back and modify the taskChangeTester() if that works better for the rest of the code. Just let me know!